scipy/stats/_continuous_distns.py

```diff
@@ -3392,6 +3392,37 @@ def _cdf(self, x, mu):
     def _stats(self, mu):
         return mu, mu**3.0, 3*np.sqrt(mu), 15*mu
 
+    def fit(self, data, *args, **kwds):
+
+        if type(self) == wald_gen:
+            return super(invgauss_gen, self).fit(data, *args, **kwds)
+
+        data, fshape_s, floc, fscale = _check_fit_input_parameters(self, data,
+                                                                   args, kwds)
+        '''
+        Source: Statistical Distributions, 3rd Edition. Evans, Hastings,
+        and Peacock (2000), Page 121. Their shape parameter is equivilent to
+        SciPy's with the conversion `fshape_s = fshape / scale`.
+
+        MLE formulas are not used in 3 condtions:
+        - `floc` is not fixed
+        - `f0` is fixed
```

Comment:
I believe `f0` is mu for invgauss. Should we use it by name here, maybe 
`- 'mu' (fshape[0]) is fixed.`

Reply:
True, it would probably be more precise to say,

```
- `loc` is not fixed
- `mu` is fixed
```
since `floc` already seems to imply "fixed loc", and we don't want to say "fixed loc is fixed".